data/basics/units.go

```diff
@@ -130,3 +130,8 @@ func (round Round) SubSaturate(x Round) Round {
 
 	return round - x
 }
+
+// RoundUpToMultipleOf rounds up round to the next multiple of n.
+func (round Round) RoundUpToMultipleOf(n Round) Round {
```

Comment:
I'm not sure if `n` needs to be a `basics.Round` here. But I can't see any concrete harm done here.